src/3d/qgscameracontroller.h

```diff
@@ -229,8 +229,17 @@ class _3D_EXPORT QgsCameraController : public QObject
     {
       None = 0,
       Translation,
-      Rotation,
-      Zoom
+      RotationCamera,
+      RotationCenter,
+      Zoom,
+      ZoomWheel
```

Comment:
Can you add comment to explain, for each entry, what operation/combo-key/mouse-click matches?

Reply:
Done.